espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/GetClipboard.kt

```diff
@@ -33,6 +33,13 @@ class GetClipboard : RequestHandler<GetClipboardParams, String?> {
 
     @Throws(AppiumException::class)
     override fun handleInternal(params: GetClipboardParams): String? {
+        // Can be null if contentType was no plaintext
+        if (params.contentType == null
```

Comment:
I assume that it should be set to 'plaintext' be default if nothing else was set

Reply:
Yes, I also assumed so, but debugger showed here was `null` for `{"contentType":"plaintext","content":"aGFwcHkgdGVzdGluZw==","label":"Note"}`
Maybe, something was wrong in Gson mapping.
(would like to dig further as after this fix)